var/spack/repos/builtin/packages/py-openmc/package.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyOpenmc(PythonPackage):
+    """The OpenMC project aims to provide a fully-featured Monte Carlo particle
+       transport code based on modern methods. It is a constructive solid
+       geometry, continuous-energy transport code that uses ACE format cross
+       sections. The project started under the Computational Reactor Physics
+       Group at MIT."""
+
+    homepage = "http://openmc.readthedocs.io/"
+    url = "https://github.com/openmc-dev/openmc/tarball/v0.10.0"
+    git = "https://github.com/openmc-dev/openmc.git"
+
+    version('develop')
+    version('master')
+    version('0.12.0', tag='v0.12.0', submodules=True)
+    version('0.11.0', sha256='19a9d8e9c3b581e9060fbd96d30f1098312d217cb5c925eb6372a5786d9175af')
```

Comment:
Why not use a url/checksum for both versions?

Reply:
I didn't use the url/checksum for version 0.12.0 because OpenMC switched to using submodules which aren't present in the URL tarball.